kura/examples/org.eclipse.kura.wire.devel.component.provider/src/org/eclipse/kura/wire/devel/DataTypeHelper.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ *******************************************************************************/
+package org.eclipse.kura.wire.devel;
+
+import java.util.Base64;
+import java.util.Base64.Decoder;
+
+import org.eclipse.kura.type.DataType;
+import org.eclipse.kura.type.TypedValue;
+import org.eclipse.kura.type.TypedValues;
+
+public final class DataTypeHelper {
+
+    private static final Decoder decoder = Base64.getDecoder();
+
+    private DataTypeHelper() {
```

Comment:
You could also throw IllegalAccessError since private constructors can also be invoked using reflection